ha_vpn_over_interconnect_5GB_attach/main.tf

```diff
@@ -0,0 +1,295 @@
+# HA VPN over Cloud Interconnect 5GB VLAN attachments (Dedicated Interconnect)
+
+# [START cloudinterconnect_ha_vpn_over_interconnect_5gb_attachments]
+
+# Create all resources in the same region, which you can specify at the provider-level
+provider "google" {
+  project = "your-project"
```

Comment:
Remove

Reply:
OK, let's keep it but comment it out so that copy/paste/run in Cloud Shell works.